provisioning/miniprov/hare_mp/main.py

```diff
@@ -565,6 +565,14 @@ def add_param_argument(parser):
     return parser
 
 
+def add_factory_argument(parser):
+    parser.add_argument('--pre-factory',
+                        help='Pre-factory will take you to before-factory',
```

Comment:
The help message is not very clear, `pre-factory` is synonymous with `before-factory`. Consider,
```
Deletes contents of /var/log/seagate/hare, /var/lib/hare and /var/motr. Undoes everything that is done in post-install stage of hare mini-provisioner.
```

Reply:
Let's define a stub for `pre-factory` option and invoke it from cleanup() if `--pre-factory` argument is passed to cleanup command.

Reply:
Make sure that the pre-factory stub is invoked if `--pre-factory` argument is passed to cleanup command.

Reply:
Added new stub for `pre-factory` and invoked it from cleanup() if `--pre-factory` argument is passed to cleanup command.

Reply:
> The help message is not very clear, `pre-factory` is synonymous with `before-factory`. Consider,
> 
> ```
> Deletes contents of /var/log/seagate/hare, /var/lib/hare and /var/motr. Undoes everything that is done in post-install stage of hare mini-provisioner.
> ```

Added this content in the help message.